mix.exs

```diff
@@ -5,14 +5,14 @@ defmodule Delta.MixProject do
     [
       app: :delta,
       version: "0.1.0",
-      elixir: "~> 1.9",
+      elixir: "~> 1.11",
       elixirc_paths: elixirc_paths(Mix.env()),
       compilers: [:phoenix] ++ Mix.compilers(),
       start_permanent: Mix.env() == :prod,
       deps: deps(),
       releases: releases(),
       dialyzer: [
-        plt_add_deps: :transitive,
+        plt_add_deps: :app_tree,
```

Comment:
For this project, what's the difference between these options? I'm not sure how this relates to the commit that introduces it, which is the "update Elixir/Erlang" one.

Reply:
There's a warning that started appearing (or at least, I started noticing it):
```
Dialyxir has deprecated plt_add_deps: :transitive in favor of app_tree, which includes only runtime dependencies.
```

Reply:
Ah, nice, makes sense then!